### PR TITLE
Ui assessment amhp feedback

### PR DESCRIPTION
**Pull request by stephenharland**

### CODE REVIEW

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html [hunk 1]
```diff
@@ -80,6 +80,9 @@ <h6 class="card-subtitle mb-2 text-muted">
             </div>
             <div class="form-row" *ngIf="amhpField.errors">
               <div class="col-md-8 mb-2">
+                <div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
+                  <span>* No Results Found </span>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html]: <span> is not required here

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.ts [hunk 2]
```diff
@@ -621,4 +622,11 @@ export class AssessmentCreateComponent implements OnInit {
       this.toBeCompletedByDateField.setErrors(null);
     }
   }
+
+  ValidateTypeAheadResults(results: any[], fieldName: string) {
+
+    if (results == null) {
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.ts]: Best to use ===

File: Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html [hunk 3]
```diff
@@ -51,6 +51,9 @@ <h6 class="card-subtitle mb-2 text-muted">
             </div>
             <div class="form-row" *ngIf="amhpField.errors">
               <div class="col-md-8 mb-2">
+                <div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
+                  <span>* No Results Found </span>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html]: span not required

File: Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.ts [hunk 4]
```diff
@@ -694,4 +695,11 @@ export class AssessmentEditComponent implements OnInit {
 
     return formIsValid;
   }
+
+  ValidateTypeAheadResults(results: any[], fieldName: string) {
+
+    if (results == null) {
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.ts]: Best to use ===

Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.ts]: replaced == with ===

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.ts (hunk 2 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.ts]: replaced == with ===

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html (hunk 1 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html]: removed error message

File: Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html (hunk 3 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html]: removed error message

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html [hunk 5]
```diff
@@ -80,9 +80,6 @@ <h6 class="card-subtitle mb-2 text-muted">
             </div>
             <div class="form-row" *ngIf="amhpField.errors">
               <div class="col-md-8 mb-2">
-                <div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
-                  <span>* No Results Found </span>
-                </div>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html]: mis-communication here:
The div is required but the span tags around "* No Results Found" are not required i.e. we need:
<div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
    * No Results Found
</div>

File: Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html [hunk 6]
```diff
@@ -51,9 +51,6 @@ <h6 class="card-subtitle mb-2 text-muted">
             </div>
             <div class="form-row" *ngIf="amhpField.errors">
               <div class="col-md-8 mb-2">
-                <div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
-                  <span>* No Results Found </span>
-                </div>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html]: mis-communication here:
The div is required but the span tags around "* No Results Found" are not required i.e. we need:
<div *ngIf="amhpField.errors.NoMatchingResults" class="invalid-field">
    * No Results Found
</div>

File: Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html (hunk 5 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-create/assessment-create.component.html]: re-added error message and removed span tag

File: Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html (hunk 6 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/assessment/assessment-edit/assessment-edit.component.html]: re-added error message and removed span tag

**Pull request merged**